### PR TITLE
Feat/split snv sv indel

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Added:
- A new rule to split SV/SNV calls from Sentieon TNScope output

Changed:
- Merged TNScope modelapply into TNScope variant calling rule for tumor-normal WGS

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by hassanfa** [comment]
tumor only WGS split:
image

tumor-normal WGS split:
image

File: BALSAMIC/snakemake_rules/variant_calling/sentieon_tn_varcall.rule [hunk 1]
```diff
@@ -110,13 +110,16 @@ rule sentieon_TNscope:
...
-        namemap = vcf_dir + "SNV.somatic." + config["analysis"]["case_id"] + ".tnscope.sample_name_map",
+        vcf_all = vcf_dir + "sentieon_tnscope/ALL.somatic." + config["analysis"]["case_id"] + ".tnscope.vcf.gz",
+        tnscope_filtered_vcf = vcf_dir + "sentieon_tnscope/ALL_filtered.somatic." + config["analysis"]["case_id"] + ".tnscope.vcf.gz",
+        namemap_snv = vcf_dir + "SNV.somatic." + config["analysis"]["case_id"] + ".tnscope.sample_name_map",
+        namemap_sv = vcf_dir + "SV.somatic." + config["analysis"]["case_id"] + ".tnscope.sample_name_map",
     params:
         tmpdir = tempfile.mkdtemp(prefix=tmp_dir),
         tumor = get_sample_type(config["samples"], "tumor"),
         normal = get_sample_type(config["samples"], "normal"),
         variant_setting = "--min_init_normal_lod 0.5 --min_normal_lod 1.0 --min_init_tumor_lod 1.0 --min_tumor_lod 8",
```
Comment (ashwini06) [inline BALSAMIC/snakemake_rules/variant_calling/sentieon_tn_varcall.rule:121]: can we move this to constants?

**Review by ashwini06 — COMMENTED** [review]
Looks good 💯  👍

**Review by ashwini06 — COMMENTED** [review]
Looks good 👍

**Pull request merged**